src/app/campaign.service.ts

```diff
@@ -56,11 +56,11 @@ export class CampaignService {
     let dateToUse: Date|undefined;
 
     if (this.isInFuture(campaign)) {
-      dateToUse = campaign.startDate;
```

Comment:
this is a runtime change - now we actually return a Date object from here at runtime instead of a string. But it shouldn't matter, it gets passed to Angular's DatePipe that can handle either.